packages/cashscript/src/Argument.ts

```diff
@@ -56,15 +56,15 @@ export function encodeArgument(
 
   // Redefine SIG as a bytes65 so it is included in the size checks below
   // Note that ONLY Schnorr signatures are accepted
-  if (type === PrimitiveType.SIG) {
+  if (type === PrimitiveType.SIG && argument.byteLength !== 0) {
```

Comment:
Is this a gap fixed in 0.6.5 also?

Reply:
Tbh I'm not entirely sure any more what this change is good for specifically. But no, this change won't be ported to 0.6.5.